msu/hooks/items/weapons/weapon.nut

```diff
@@ -33,6 +33,17 @@
 		}
 	}
 
+	local addSkill = o.addSkill;
+	o.addSkill = function( _skill )
+	{
+		addSkill(_skill);
+		if (::MSU.isIn("AdditionalAccuracy", _skill))
+		{
+			_skill.m.AdditionalAccuracy += this.m.AdditionalAccuracy;
+			_skill.setBaseValue("AdditionalAccuracy", _skill.m.AdditionalAccuracy);
+		}
+	}
```

Comment:
return the result of wrapped addSkill, even if it doesn't return anything rn it's best practice